gemfile.js

```diff
@@ -8,6 +8,7 @@ const GEMFILE_DEFAULT_LOCATION = path.resolve(process.cwd(), 'Gemfile.lock');
 const WHITESPACE = /^(\s*)/;
 const GEMFILE_KEY_VALUE = /^\s*([^:(]*)\s*\:*\s*(.*)/;
 const ORIGINS = ['GEM', 'GIT', 'PATH'];
+const RUBY = /^ruby\s*(.*)/;
```

Comment:
Are you sure with `\s*`? It will catch "ruby               something" as well as "rubysomethingelse". Not sure what are the possible values. Maybe you want `\s?` (zero or one).